Documentation/user-guides/running-exporters.md

```diff
@@ -96,11 +96,142 @@ By default, the `PodMonitor` and `ServiceMonitor` objects include runtime metada
+
+```yaml
+- action: replace
+  replacement: prometheus
+  targetLabel: team
+```
+
+**Filtering targets by label**
+
+The following snippet will configure Prometheus to scrape metrics from the targets if they have the `team` label set to `prometheus` and `datacenter` not set to `west_europe`.
+
+```yaml
+- sourceLabels:
+  - team
+  regex: "prometheus"
+  action: keep
+- sourceLabels:
+  - datacenter
+  regex: west_europe
+  action: drop
+```
+
+**Full example**
+
+The following snippet is using a `ServiceMonitor` to only select targets that have the `team` label set to `prometheus` and exclude the ones that have `datacenter` set to `west_europe`. The same configuration may be used with a `PodMonitor`.
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: example-app
+  labels:
+    team: frontend
+spec:
+  selector:
+    matchLabels:
+      app: example-app
+  endpoints:
+  - port: web
+    relabelings:
+      - sourceLabels:
+          - team
+        regex: "prometheus"
+        action: keep
+      - sourceLabels:
+          - datacenter
+        regex: west_europe
+        action: drop
+```
+
+#### Metric Relabeling
+
+Metric relabeling is applied to samples as the last step before ingestion, and it has the same configuration format and actions as target relabeling.
+
+> Metric relabeling does not apply to automatically generated timeseries such as up.
+
+**Dropping metrics**
+
+The following configuration drops any metric which name (`__name__`) matches the regex `container_tasks_state`.
+
+```yaml
+metricRelabelings:
+- sourceLabels:
+  - __name__
+  regex: container_tasks_state
+  action: drop
+```
+
+**Dropping time series**
+
+The following configuration drops metrics where the `id` label matches the regex `/system.slice/var-lib-docker-containers.*-shm.mount`.
+
+```yaml
+metricRelabelings:
+- sourceLabels:
+  - id
+  regex: '/system.slice/var-lib-docker-containers.*-shm.mount'
+  action: drop
+```
+
+**Full example**
+
+The following `PodMonitor` configures Prometheus to drop metrics where the `id` label matches the regex `/system.slice/var-lib-docker-containers.*-shm.mount`. The same configuration could also be used with a `ServiceMonitor`
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: example-app
+  labels:
+    team: frontend
+spec:
+  selector:
+    matchLabels:
+      app: example-app
+  endpoints:
+  - port: web
+    metricRelabelings:
+    - sourceLabels:
+      - id
+      regex: '/system.slice/var-lib-docker-containers.*-shm.mount'
+      action: drop
+```
+
+For more information about `relabeling` and `metricRelabelings` please check the official Prometheus document.
+
+- [Relabel Config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config)
+- [Metric Relabel Config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#metric_relabel_configs)
+
 ### Configuration
```

Comment:
do we still need this section?

Reply:
I see this as backlinks, don't add anything extra but make the user's life easier.
@simonpasquier should I keep or drop?

Reply:
@simonpasquier wdyt? should I keep or drop it?

Reply:
I would drop this section as it's redundant with your addition (unless I miss something).

Reply:
@simonpasquier agreed, I removed it.
Can you give a last review?

Reply:
Hmm I don't see the removal in the PR. Did you forget to push?
To be clear, we should remove L225 to L236.

Reply:
Sorry!
Miss the push, now I guess this is right